static/src/javascripts/projects/common/modules/article/flyers.js

```diff
@@ -51,7 +51,7 @@ define([
             minBelow: 250,
             selectors: {
                 ' > h2': {minAbove: detect.getBreakpoint() === 'mobile' ? 20 : 0, minBelow: 200},
-                ' > *:not(p):not(h2)': {minAbove: 35, minBelow: 300},
+                ' > *:not(p):not(h2):not(blockquote)': {minAbove: 35, minBelow: 300},
```

Comment:
This selector is getting **Cr4$z%y**, out-of-interest,  do we even know if this works in all browser implementations of `querySelectorAll` ?